src/models/vehicle/Chassis.tsx

```diff
@@ -102,8 +102,10 @@ export const Chassis = forwardRef<Object3D, PropsWithChildren<BoxProps>>(({ args
     })
   }, [maxSpeed])
 
+  let camera: string
```

Comment:
```suggestion
  let camera: Camera
```

Reply:
isn't the camera stored in the store type string tho? 
https://github.com/pmndrs/racing-game/blob/main/src/store.ts#L12

Reply:
It's not just any string, it's a string literal